run-mlreco/install_mlreco.sh

```diff
@@ -92,11 +92,9 @@ cd ..
 # git clone -b jw_dune_nd_lar https://github.com/chenel/lartpc_mlreco3d.git
 
 #git clone -b v2.9.5 https://github.com/DeepLearnPhysics/lartpc_mlreco3d.git
-git clone -b v0.4.3 https://github.com/DeepLearnPhysics/spine.git
+git clone -b develop https://github.com/DeepLearnPhysics/spine.git
 
 # git clone https://github.com/chenel/dune-nd-lar-reco.git
 # the old yaml.load API has been removed
 # sed -i 's/yaml.load(open(filename))/yaml.load(open(filename), yaml.Loader)/' \
 #     dune-nd-lar-reco/load_helpers.py
-
-git clone -b main https://github.com/DeepLearnPhysics/spine_prod.git
```

Comment:
This was introduced on purpose in PR #47 so that the spine jobs can point to `spine_prod` for the configs (which is where they live officially) rather than doing a manual copy from there to `configs` whenever there is a new one.